app/views/admin/api_docs/base/index.html.slim

```diff
@@ -9,7 +9,7 @@
 - if @api_docs_services.any?
   = render partial: 'active_docs_table', locals: {service: @service, api_docs_services: @api_docs_services}
 - else
-  - if current_account.api_docs_services.without_service.any? && can?(:manage, :portal)
+  - if current_account.api_docs_services.without_owner.any? && can?(:manage, :portal)
```

Comment:
I think this is contradictory. `api_docs_services` is now defined as with `service_id` present, while `api_docs_services` looks for the opposite.

Reply:
Also, let's keep in mind that sometimes we want all ApiDocs owned directly by the `Account`, i.e. NOT owned by a `Service`, nor a `BackendApi`; and sometimes we want ALL ApiDocs, i.e. the ones owned by the `Account` directly, by the `Service`'s and `BackendApi`'s of the account.

Perhaps `Account` could have `has_many :api_docs` and `has_many :all_api_docs` maybe? (Or `:api_docs_owned_by_self` and `:api_docs` 🤔